server/graph/resolvers/asset.js

```diff
@@ -20,18 +20,29 @@ module.exports = {
       if (args.kind !== 'ALL') {
         cond.kind = args.kind.toLowerCase()
       }
-      const result = await WIKI.models.assets.query().where(cond)
-      return result.map(a => ({
+      const folderHierarchy = await WIKI.models.assetFolders.getHierarchy(args.folderId)
+      const folderPath = folderHierarchy.map(h => h.slug).join('/')
+      let results = await WIKI.models.assets.query().where(cond)
+      results = _.filter(results, r => {
```

Comment:
Don't reassign `results` here. Should be `const results = ...` then you return the `_filter().map()` as 1 chained statement.